clients/drcachesim/CMakeLists.txt

```diff
@@ -168,6 +168,11 @@ add_exported_library(drmemtrace_func_view STATIC tools/func_view.cpp)
 add_exported_library(drmemtrace_invariant_checker STATIC tools/invariant_checker.cpp)
```

Comment:
It would be a lot easier to review this prototype code with a good overview and description.  Please edit the PR description to describe the components being added here and what each one does.